overlays/haskell.nix

```diff
@@ -471,7 +471,15 @@ final: prev: {
                   inherit (rev) sha256;
                 };
                 revSuffix = final.lib.optionalString (rev.revNum > 0) "-r${toString rev.revNum}";
-            in let src = final.buildPackages.pkgs.runCommand "${name}-${version'}${revSuffix}-src" { } (''
+            in let src = final.buildPackages.pkgs.runCommand "${name}-${version'}${revSuffix}-src" {
+                passthru = {
+                  # TODO remove once nix >=2.4 is widely adopted (will trigger rebuilds of everything).
```

Comment:
Bit sad that we have to do this, but probably a good idea. Maybe add a pointer to this from the note in `clean-source-with` so we remember it exists?

Reply:
Comment added.